src/generator/WDL/entities/WorkflowGenerator.js

```diff
@@ -8,7 +8,14 @@ import { buildDeclarations } from '../utils/utils';
 
 export default class WorkflowGenerator {
   constructor(wfStep, settings) {
+    this.wfBodyElementGenerators = {
+      call: this.genCall,
+      scatter: this.genScatter,
+      whileloop: this.genWhile,
+      if: this.genIf,
+    };
```

Comment:
Thank you so much for deep analysis of my changes, it is really helpfull for me to get so detailed review!

This name threated by this way because **hermes\wdl_parser.js** named this so. 

Please look at the endings of the grammar rules listed below: (there are names the same to my processor names mapping array)
**grammar.hgr**
```
    $while_loop = :while :lparen $e :rparen :lbrace list($wf_body_element) :rbrace -> WhileLoop(expression=$2, body=$5)
    $if_stmt = :if :lparen $e :rparen :lbrace list($wf_body_element) :rbrace -> If(expression=$2, body=$5)

    $scatter = :scatter :lparen :identifier :in $e :rparen :lbrace list($wf_body_element) :rbrace
        -> Scatter(item=$2, collection=$4, body=$7)
```

If this conduct could not be a reason of some issues related to this naming, please let me to leave it.